.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="MagicConstant" enabled="false" level="WARNING" enabled_by_default="false" />
```

Comment:
Ah, looks like IntelliJ was yelling at you for putting a bunch of test data huh? There is probably a neater way than just disabling this warning for the entire project - it is a good warning in general. Let's delete this file and see if we can make it so that only that one field in the seed data is permitted without warnings.